File: main/src/main/java/net/fwitz/math/main/complex/analysis/zeta/XiPlot.java
package net.fwitz.math.main.complex.analysis.zeta;

import net.fwitz.math.complex.analysis.RiemannZeta;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class XiPlot {
    public static void main(String[] args) {
        new ComplexFunctionPlot("xi(s)")
                .domainX(-5, 5)
                .domainY(-5, 5)
                .size(500, 500)
                .computeFn(RiemannZeta::xi)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/analysis/zeta/XiPlot.kt
package net.fwitz.math.main.binary.complex.analysis.zeta

import net.fwitz.math.binary.complex.analysis.RiemannZeta
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object XiPlot {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("xi(s)")
        .domainX(-5.0, 5.0)
        .domainY(-5.0, 5.0)
        .size(500, 500)
        .computeFn(RiemannZeta::xi)
        .render()
}